### DISCUSSION TITLE
请问怎么把老的单词导入到anki

### DISCUSSION THREAD

**Discussion by ningyuwhut** [crimx/ext-saladict#1983] [Q&A]
我按照教程（https://saladict.crimx.com/anki.html ）设置了anki，新加入单词本的单词可以导入到anki中了，但是老的单词好像没有导入到anki，这个该怎么操作呢。
按照教程我把导出模板写成了下面这样：
```
%date%`%text%`%trans%`%context%`%contextCloze%`%note%`%title%`%url%`%favicon%`%audio%
```
但是在anki中导入时出现如下提示，但是其实并没有导入成功
```
导入成功。
文件中出现两次：⁨3/2/2023⁩
文件中出现两次：⁨3/2/2023⁩
文件中出现两次：⁨3/2/2023⁩
文件中出现两次：⁨3/2/2023⁩
文件中出现两次：⁨3/2/2023⁩
文件中出现两次：⁨3/2/2023⁩
文件中出现两次：⁨3/2/2023⁩
文件中出现两次：⁨3/2/2023⁩
文件中出现两次：⁨3/2/2023⁩
已添加⁨0⁩条笔记, 已更新⁨0⁩条笔记, ⁨10⁩ 条笔记未变更.
首字段匹配的: ⁨3/2/2023⁩
首字段匹配的: ⁨3/1/2023⁩
首字段匹配的: ⁨2/28/2023⁩
首字段匹配的: ⁨2/24/2023⁩
首字段匹配的: ⁨2/22/2023⁩
首字段匹配的: ⁨2/21/2023⁩
首字段匹配的: ⁨2/15/2023⁩
首字段匹配的: ⁨2/14/2023⁩
首字段匹配的: ⁨2/13/2023⁩
首字段匹配的: ⁨2/12/2023⁩
```

**Comment by crimx** [comment]
说明之前应该已经成功导入进去了。后面手动导入都是重复的，Anki 跳过了。

**Reply by ningyuwhut** [reply]
上次我又看了下，其实并没有导入进去，因为在设置自动同步到anki之前的那些单词的Date字段是”3/2/2023⁩“这种格式的，而导入的时候好像是Date字段相同的单词只会导入第一个，所以会出现”文件中出现两次：⁨3/2/2023⁩”这种提示。后来我把所有这些单词的Date字段手动改成了时间戳格式，保证每个单词的Date字段是不重复的，再次导入就没有问题了。 可以的话麻烦帮忙看下这里是否可以把单词的Date字段默认使用时间戳，这样就不会出现这个问题了

**Reply by ningyuwhut** [reply]
还有一个问题，这两天发现利用上面的方式导入到anki中的单词卡片没法显示Context 和ContextCloze两个字段，也就是背单词的时候看不到句子。我摸索了下，发现原因是ContextCloze中目标单词是用连续的下划线表示的，而自动同步到anki中的卡片中的ContextCloze中的目标单词是用{{c1::dizzying}} 这种格式表示的。我把______ 手动改成{{c1::dizzying}}之后卡片中的Context 和ContextCloze两个字段就可以正常显示了。请问这种现象是我设置的问题还是anki默认就是这样的呢
“'Not Just a Silly Balloon': ______ and Fear Over Another US-China Clash”
